rfcs/text/0000-authentication-fields.md

```diff
@@ -0,0 +1,155 @@
+# 0000: Authentication Fields RFC
+
+Stage: 0
+Date: 2021-08-08
+
+This RFC will provide normalization for fields related to authentication fields to assure that they are retained primarily in ECS core and in any extended fieldset when needed. These fields are important to normalize into ECS fields in order to maximize effectiveness of cross log utilization. 
+
+## Fields
+
+#### Activity Log
+|Proposed Field Name|Type|Value|Azure Field|
+| --- | --- | --- | --- |
+|authentication.scope|keyword|user_impersonation|azure.activitylogs.identity.claims.http://schemas_microsoft_com/identity/claims/scope|
+|authentication.authentication.requirement|keyword|multiFactorAuthentication|azure.activitylogs.properties.authenticationRequirement|
```

Comment:
could this just be `authentication.requirement`?

Reply:
That works

Reply:
Change has been made to make authentication.authentication.requirement to authentication.requirement